api/src/utils/xlsx-utils/worksheet-utils.ts

```diff
@@ -321,3 +321,28 @@ export function validateCsvFile(xlsxWorksheet: xlsx.WorkSheet, columnValidator:
 
   return true;
 }
+
+/**
+ * This function pulls out any non-standard columns from a CSV so they can be processed separately.
+ *
+ * @param {xlsx.WorkSheet} xlsxWorksheet The worksheet to pull the columns from
+ * @returns {*} string[] The list of non-standard columns found in the CSV
+ * //TODO: Remove old reference to this file standard-column-utils
+ */
+export function getNonStandardColumnNamesFromWorksheet(
```

Comment:
Note to self: Update observations import code to use this instead of (standard-column-utils) version. This version now accepts the validator as a param.